waf.go

```diff
@@ -76,15 +76,24 @@ func NewWAF(config WAFConfig) (WAF, error) {
 		waf.ContentInjection = true
 	}
 
-	if r := c.requestBody; r != nil {
+	if c.requestBodyAccess {
 		waf.RequestBodyAccess = true
-		waf.RequestBodyLimit = int64(r.limit)
-		waf.RequestBodyInMemoryLimit = int64(r.inMemoryLimit)
 	}
 
-	if r := c.responseBody; r != nil {
+	if c.requestBodyLimit != 0 {
```

Comment:
It is okay to assume that `0` is the default value and not a value that the user want to enforce? Otherwise I'm going to initially populate it inside the `NewWAFConfig` constructor adding an invalid value like `-1`

Reply:
-1 sounds better cc @anuraaga @jptosso

Reply:
Thanks, done!